### DISCUSSION TITLE
PerlinAreaDeform with calc normals

### DISCUSSION THREAD

**Discussion by ikekou** [cables-gl/cables_docs#205] [Answers & Questions] [answered]
Hello. This is my first post here.

Now, I have a question. It is about PerlinAreaDeform OP.

I have placed a Sphere and deformed it with PerlinAreaDeform. I am placing one light.

When I turn on Calc Normals, the direction that the light is hitting is reversed. If I had to guess, I would say that maybe the Normals are reversed? I don't know.

I want the light to shine from the right direction. Please let me know if you have any solutions.

https://cables.gl/p/6I7ZYX

<img width="1200" alt="Screen_Shot_2022-02-16_at_16 24 59" src="https://user-images.githubusercontent.com/458137/158909606-b9a44674-5112-435e-aa6e-f22709ebe94c.png">

<img width="1198" alt="Screen_Shot_2022-02-16_at_16 25 11" src="https://user-images.githubusercontent.com/458137/158909616-81e25b24-d7b6-4c47-879d-8e2d568d31ff.png">

**Comment by pandrr** [comment, marked as answer]
https://github.com/cables-gl/cables_docs/issues/208
